citadel/ros_integration.md

```diff
@@ -24,7 +24,14 @@ to verify if your message type is supported by the bridge.
 **Not available at the moment**
 
 The repository `packages.osrfoundation.org` hosts binary packages for Blueprint
-but for Citadel the recommended method is to build from source
+but for Citadel the recommended method is to build from source.
```

Comment:
Nice! Thanks for staying on top of this! Do you mind also removing the "Not available" notice and this paragraph, and substituting with a link to this README which summarizes what combinations are available and how?

https://github.com/ignitionrobotics/ros_ign